packages/ui/src/primitives/Card.tsx

```diff
@@ -0,0 +1,13 @@
+import { styled } from '../../stitches.config'
```

Comment:
Looking at this component, is it reusable or does this just seem like a box with css? Maybe we can just move it to live within the AttributeSelect as a box with some css until it pops up in another design.